tools/commands/run.cpp

```diff
@@ -17,6 +18,45 @@ constexpr auto create_address = 0xc9ea7ed000000000000000000000000000000001_addre
 
 /// The gas limit for contract creation.
 constexpr auto create_gas = 10'000'000;
+
+auto bench(MockedHost& host,
+           evmc::VM& vm,
+           evmc_revision rev,
+           const evmc_message& msg,
+           bytes_view code,
+           const evmc::result& expected_result,
+           std::ostream& out)
+{
+    {
+        using clock = std::chrono::steady_clock;
+        using unit = std::chrono::nanoseconds;
+        constexpr auto unit_name = " ns";
+        constexpr auto target_bench_time = std::chrono::seconds{1};
+        constexpr auto warning =
+            "WARNING! Inconsistent execution result likely due to usage of storage ";
```

Comment:
I think these may be better, but who knows:
- `the usage of storage`
- `the use of storage`